pkg/apis/serving/v1alpha1/testing/configuration_builder.go

```diff
@@ -0,0 +1,144 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	buildv1alpha1 "github.com/knative/build/pkg/apis/build/v1alpha1"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clientgotesting "k8s.io/client-go/testing"
+)
+
+type ConfigToBuildFunc func(*v1alpha1.Configuration) *buildv1alpha1.Build
+type ConfigToRevisionFunc func(*v1alpha1.Configuration) *v1alpha1.Revision
+
+type ConfigBuilder struct {
+	obj *v1alpha1.Configuration
+}
+
+func Config(name, namespace string) ConfigBuilder {
```

Comment:
I'm going to swap this to be `(namespace, name string)` to be consistent with revision & route table tests